.eslintrc.json

```diff
@@ -520,6 +520,7 @@
       "never"
     ],
     "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
Why this rule here?

Reply:
```typescript
$checkWithinAccessParams!(pos, 0, this.length - 1);
```
null assertion is required when calling a macro.